### DISCUSSION TITLE
are pmap(f) and pmap(jit(f)) effectively the same?

### DISCUSSION THREAD

**Discussion by matpalm** [jax-ml/jax#5569] [Q&A] [answered]
wondering when would they ever have a different result?

trying to get a better understanding of the `pmap` method comment...

```
Applying pmap() to a function will compile the function with XLA (similarly to jit())
```

**Comment by gnecula** [comment, marked as answer]
Functionally, a `jit` inside a `pmap` will not have any effect. The only visible effect is that a `jit` will add a name scope around its computation so you may be able to better identify a computation in a profile. However, a clearer way to achieve that is with `jax.named_call`.
